tests/_support/Api.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+trait Api
+{
+    /**
+     * Perform request to API.
+     *
+     * @param string $endpoint
+     * @param string $method
+     * @param array  $data
+     *
+     * @return array|bool|mixed
+     */
+    private function request($method, $endpoint, $body = [])
+    {
+        $method    = 'GET';
```

Comment:
:thinking: aqui não seria `$method`?
```suggestion
        $method = $method;
```

Aí, talvez colocar o `GET` como default: `private function request($method = 'GET', $endpoint, $body = [])`

Reply:
trolei kkk

Reply:
Trolei :man_facepalming: